src/renderers/shared/reconciler/ReactCompositeComponent.js

```diff
@@ -195,6 +196,18 @@ var ReactCompositeComponentMixin = {
           Component.displayName || Component.name || 'Component'
         );
       }
+
+      var propsMutated = !shallowEqual(inst.props || {}, publicProps);
+      var componentName = Component.displayName || Component.name ||
```

Comment:
Yes, we prefer

```
      var componentName =
        Component.displayName || Component.name || 'Component';
```

to

```
      var componentName = Component.displayName || Component.name ||
            'Component';
```

(since it doesn't fit all on one line).